strr-platform-web/app/pages/auth/account/choose-existing.vue

```diff
@@ -0,0 +1,182 @@
+<script setup lang="ts">
+// import { isoCountriesList } from '~/utils/isoCountriesList'
+const localePath = useLocalePath()
+const { t } = useI18n()
+const isSmallScreen = useMediaQuery('(max-width: 640px)')
+const strrModal = useStrrModals()
+const accountStore = useConnectAccountStore()
+
+useHead({
+  title: t('platform.title.chooseAccount')
+})
+
+definePageMeta({
+  middleware: ['auth']
+})
+
+// display premium accounts first
+const sortedAccounts = computed(() => {
+  return accountStore.userAccounts.sort((a, b) => {
+    if (a.accountType === AccountType.PREMIUM && b.accountType !== AccountType.PREMIUM) {
+      return -1
+    } else if (a.accountType !== AccountType.PREMIUM && b.accountType === AccountType.PREMIUM) {
+      return 1
+    }
+    return 0
+  })
+})
+// const sortedAccounts = []
+
+// TODO: how do we want to redirect if the user has/does not have any platforms?
```

Comment:
always to the dashboard. If they don't have any platforms then they'll have the 'begin application' todo in there

Reply:
short-term (for now) we can redirect to the dashboard - longer term we want to direct as in the figjam flow. Zero platforms would launch the registration application. 1 platform would go to the platform dashboard. Future looking, we will need to handle multiple "objects" when we get to Strata Hotels, so 2 or more "objects" (e.g., strata hotels) would redirect to the "My STRR Dashboard (BRD) (the Host team is working on this too).